test/system/065-cp.bats

```diff
@@ -163,8 +163,8 @@ load helpers
 
     # In the second case, podman creates a directory nonesuch2, then
     # creates a file with the same name as the input file. THIS IS WEIRD!
-    run_podman exec cpcontainer cat /tmp/nonesuch2/$rand_filename2
-    is "$output" "$rand_content2" "cp creates destination dir and file"
+    run_podman 1 exec cpcontainer cat /tmp/nonesuch2/$rand_filename2
```

Comment:
Since podman behavior has changed, this is no longer a great test: it simply checks that the destination file doesn't exist - but it is conceivable (albeit improbable) that podman created the destination directory before exiting. That would be very bad. A much better test would now be:
```
# cp into nonexistent directory should not mkdir said directory
run_podman 1 exec cpcontainer test -e /tmp/nonesuch2
```
This test is much more useful.